tests/model_registry/conftest.py

```diff
@@ -227,34 +233,16 @@ def model_registry_instance(
         ) as mr:
             mr.wait_for_condition(condition="Available", status="True")
             mr.wait_for_condition(condition="OAuthProxyAvailable", status="True")
-
             yield mr
 
 
 @pytest.fixture(scope="class")
-def model_registry_mysql_config(
-    request: FixtureRequest,
-    model_registry_db_deployment: Deployment,
-    model_registry_db_secret: Secret,
-) -> dict[str, Any]:
-    """
-    Fixture to build the MySQL config dictionary for Model Registry.
-    Expects request.param to be a dict. If 'sslRootCertificateConfigMap' is not present, it defaults to None.
-    If 'sslRootCertificateConfigMap' is present, it will be used to configure the MySQL connection.
-
-    Args:
-        request: The pytest request object
-        model_registry_db_deployment: The model registry db deployment
-        model_registry_db_secret: The model registry db secret
-
-    Returns:
-        dict[str, Any]: The MySQL config dictionary
-    """
```

Comment:
should we keep the docstring? ofc updated with the new scope

Reply:
Sure. I can add it back.